### ISSUE TITLE
Stop Livestreamer

### ISSUE THREAD

**Issue by idealsat** [chrippa/livestreamer#1056]
Hi

When i play hls stream its play ok Its getting segment from http and play well till to 10 minutes.

But segment downloading slow .....example ( we have segment 1 to 50 ) when segment download 1 to 10 then its playing but 11 is downloading if its downloading slow then livestreamer timeout occur and it cant fetch segment 11 ).

Example : when download done 11 to 20 livestreamer cant read it .

How i can fix this issue.?

or 

How i can control this timeout with hls segment timeout,hls attempt etc..?

Thanks

**Comment by idealsat** [comment]
Any solution ?

**Comment by mammothb** [comment]
have you tried these cmd line options to adjust hls timeouts and other values? http://docs.livestreamer.io/cli.html#cmdoption--hls-segment-attempts

**Comment by idealsat** [comment]
yes i already try these options but with this The problem is livestreamer can read after 5 minute or 7 minute .

Even 11 to 20 downloaded before 5 minutes.

Thanks for your reply
